### ISSUE TITLE
Tag Field

### ISSUE THREAD

**Issue by PrakashParamasivam** [vjanagaran/appx#5]
Tag field background description should indicate the user as comma (',') separated

**Comment by vjanagaran** [comment]
Not clear with above details... i hope it's related to changing text

**Comment by PrakashParamasivam** [comment]
Yes. The greyed text (in the text box) should indicate the user that the tags are comma (,) separated.
